Declare iOS platform support (blocks iOS app builds: OCCTSwift requires iOS 15)

## Problem
`OCCTSwiftScripts/Package.swift` declares only:
```swift
platforms: [ .macOS(.v15) ]
```
With no `.iOS(...)` declared, SPM assumes a low default iOS minimum (12.0) for the `DrawingComposer` and `ScriptHarness` targets. But `OCCTSwift` requires **iOS 15**. Any downstream package that pulls `DrawingComposer`/`ScriptHarness` and builds for iOS fails resolution:

```
error: The package product 'OCCTSwift' requires minimum platform version 15.0 for the iOS
platform, but this target supports 12.0 (in target 'DrawingComposer' from project 'OCCTSwiftScripts')
error: ... (in target 'ScriptHarness' ...)
** BUILD FAILED **
```

This blocks building an **iOS app** that uses `DrawingComposer` (e.g. OCCTSwiftPartsAgent's SwiftUI app deploying to an iPhone). The sibling deps are already iOS-ready: OCCTSwift `.iOS(.v15)`, OCCTSwiftViewport/OCCTSwiftTools `.iOS(.v18)`.

## Fix
Add iOS (and ideally visionOS/tvOS to match the cohort) to `platforms`:
```swift
platforms: [ .macOS(.v15), .iOS(.v18) ]
```
`DrawingComposer` is geometry/DXF logic over OCCTSwift primitives (cross-platform); `ScriptHarness` is file/manifest IO. If any CLI-only code (e.g. `occtkit` process spawning) isn't iOS-friendly, keep it `#if os(macOS)`-guarded — but the **library products** (`DrawingComposer`, `ScriptHarness`) should declare iOS so downstream apps can link them.

A patch release after the bump would let consumers move their pin.

--- a/Package.swift
+++ b/Package.swift
@@ -4,7 +4,16 @@ import PackageDescription
 let package = Package(
     name: "OCCTSwiftScripts",
     platforms: [
-        .macOS(.v15)
+        .macOS(.v15),
+        // iOS so the library products (DrawingComposer, ScriptHarness) link
+        // into downstream iOS apps (e.g. OCCTSwiftPartsAgent's SwiftUI app).
+        // Floored at v18 to match the sibling cohort (Viewport/Tools/AIS
+        // are .iOS(.v18)); OCCTSwift itself only needs v15. The executable
+        // targets (occtkit, Script, the legacy Graph* tools) shell out via
+        // Foundation.Process and aren't iOS-buildable, but SPM only compiles
+        // a target when something reachable requires it — an iOS app linking
+        // a library product never pulls the executables in. See #52.
+        .iOS(.v18)
     ],
     products: [
         .library(

--- a/Sources/ScriptHarness/ScriptContext.swift
+++ b/Sources/ScriptHarness/ScriptContext.swift
@@ -45,6 +45,8 @@ public final class ScriptContext: Sendable {
     public let metadata: ManifestMetadata?
 
     public init(exportSTEP: Bool = true, metadata: ManifestMetadata? = nil) {
+        let dir: URL
+        #if os(macOS)
         let home = FileManager.default.homeDirectoryForCurrentUser
 
         // Prefer iCloud Drive for cross-device sync (Mac → iPhone)
@@ -53,12 +55,20 @@ public final class ScriptContext: Sendable {
             .appendingPathComponent("OCCTSwiftScripts/output")
         let localDir = home.appendingPathComponent(".occtswift-scripts/output")
 
-        let dir: URL
         if FileManager.default.fileExists(atPath: iCloudDir.deletingLastPathComponent().deletingLastPathComponent().path) {
             dir = iCloudDir
         } else {
             dir = localDir
         }
+        #else
+        // iOS/sandboxed platforms: homeDirectoryForCurrentUser and the
+        // ~/Library/Mobile Documents iCloud path are unavailable. Write into
+        // the app sandbox's Documents directory, which is the iCloud-syncable
+        // location on iOS when the app declares a ubiquity container.
+        let documents = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
+            ?? URL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true)
+        dir = documents.appendingPathComponent("OCCTSwiftScripts/output")
+        #endif
         self.outputDir = dir
         self.exportSTEP = exportSTEP
         self.metadata = metadata

